---
layout: "post"
title: "Understanding the Mechanism of Interaction of Bacteriophage φKZ with CRISPR-Cas Immune Systems"
author: "Anika Tiwari (‘20)"
print_publication: true
tags: issue3
categories: Perspectives
related_image: 
---

<!--excerpt-->

**Abstract:**

Bacteriophage φKZ is resistant to the DNA-targeting CRISPR-Cas9 system. Hydroxylamine mutagenesis of φKZ did not result in the weakening of the nucleus-like shell that surrounds the DNA of the bacteriophage φKZ and thus, enabled the bacteriophage φKZ to physically obstruct the Cas9 enzyme. Mutants with this desired phenotype may exist in the pool of mutant phages but were not isolated in the panel that was screened. 
The DNA sequence that contains the protospacer region of the gene gp146 of the bacteriophage φKZ is targeted by Cas13a enzyme. Without the protospacer region in the DNA to be transcribed to RNA, bacteriophage φKZ is no longer sensitive to RNA-targeting Cas13a enzyme.


<!--excerpt-->

**Objectives/Goals**

There is a continuous warfare between bacteria and bacteriophages (viruses) that infect them. Bacteria have developed CRISPR-Cas immune systems to protect themselves from invading bacteriophages. The CRISPR system uses acquired immunity, taking the DNA of bacteriophages that have previously infected it and remembering them for future infections. Upon infection, the stored DNA sequence is then cut and removed from the rest of the gene sequence, making the CRISPR system a power tool for gene-editing. Genes surrounding the CRISPR system are known as Cas, CRISPR Associated Systems, and the Type II CRISPR is found exclusively in bacteria. However, bacteriophages are unique in their interactions with the CRISPR system. Some bacteriophages such as φKZ are highly unpredictable in their actions, and their features are just being discovered. In the war between bacteria Pseudomonas aeruginosa and jumbo bacteriophage φKZ, it appears that φKZ is winning. φKZ has developed its own form of protection from the CRISPR-Cas system, resisting many DNA-targeting bacterial immune systems including CRISPR-Cas3, Cas9, Cas12, and restriction-modification systems. It has been shown that φKZ forms a nucleus-like shell around its DNA, and nucleases such as Cas9 are physically obstructed by the shell (S. D. Mendoza). However, φKZ is sensitive to an RNA-targeting CRISPR-Cas enzyme, Cas13a. The main goal of this project was to understand the mechanism by which bacteriophage φKZ interacts with different CRISPR-Cas systems. The specific aim was to identify φKZ mutants which are sensitive to DNA-targeting by Cas9 and mutants that are resistant to RNA-targeting by Cas13a. A chemical mutagenesis approach was used to create mutations in the DNA sequence of φKZ. The mutant φKZ bacteriophages (phages) were then screened to determine if any of the mutants could be defeated by Cas9 and for escapers of the RNA-targeting CRISPR-Cas enzyme, Cas13a. The mutant phages that escaped the Cas13a enzyme were sequenced to understand how they avoid Cas13a recognition. 

**Methods/Materials**

Bacteriophage φKZ was mutated with hydroxylamine, which is known to cause a non-reversible single nucleotide mutation of cytosine to thymine. The mutagenesis was conducted as described in R. Villafane (2009)’s paper. Mutated φKZ was plated on a Pseudomonas aeruginosa strain plate (PA101). The φKZ were then isolated with a toothpick and re-plated onto bacterial strain SDM065 (Cas9, II-A cRNA-φKZ) and the corresponding control strain SDM056 (Cas9, II-A cR9NA-JBD30). The bacterial strain named SDM065 or SDM056 are both Cas9 systems that target the type II-A CRISPR-RNA (cRNA) of the phages φKZ or JBD30, respectively. These two strains tested the sensitivity of mutated phages against Cas9 enzyme. SDM056 serves as the control, as it targets the cRNA of JBD30, a phage not used in this project. The phage will form a plaque no matter the mutation’s success. As SDM065 targets the cRNA of φKZ, a phage typically resistant to the CRISPR system, it would be revealed if the mutation had created a weakness in φKZ as φKZ would plaque if the mutation was successful.
Additionally, φKZ mutants were tested for temperature sensitivity as well by incubating the plates at the elevated temperature of 42°C compared to 30°C for the control group. Some phage DNA becomes sensitive to mutations at high temperatures. The high temperature activates the mutation, creating more opportunities for mutations that show sensitivity to the CRISPR system.
For identifying escapers of cas13a RNA-targeting, mutant phages were directly plated onto bacterial strain SDM078 that tested sensitivity towards RNA-targeting Cas13a enzyme system. Those that grew despite φKZ’s typical vulnerability to the CRISPR system were identified as escapers. While mutant phages were tested for their escaper tendencies, the natural or wild-type bacteriophages were tested as well. The phages that escaped targeting were purified and their CRISPR-targeting protospacer region was amplified using PCR and subsequently sequenced. 

**Results**

A thousand-fold decrease in titer value of bacteriophage φKZ was observed after mutagenesis with 0.4 M hydroxylamine for forty-eight hours. This solution of mutated φKZ was used for all subsequent experiments. One-thousand five-hundred-forty mutants from hydroxylamine mutagenesis were screened for both Cas9 and temperature sensitivity. The Cas9 sensitivity was tested by plating each mutant onto bacterial strain SDM065, designed to target the head gene gp146 of φKZ. None of mutants were found to be sensitive to the Cas9 enzyme. 

<!--excerpt-->

| ![](/imgs/systems1.png) | 
|:--:| 
|**Figure 1.** A thousand-fold decrease in titer value of bacteriophage φKZ was observed before and after forty-eight hours of hydroxylamine mutagenesis. This supports the efficiency of mutagenesis as it has a large decimating effect on a proportion of the phages. The resulting phages have either been unharmed by the mutagenic process or simply mutated (yet still resistant to targeting).

Four temperature sensitive mutants were found upon incubation of mutant phages plated onto PA01 (Pseudomonas aeruginosa strain used) at 42°C. Temperature sensitive mutants failed to grow on PA01 at the higher temperature of 42°C. The mutations inhibit the phage replication at higher temperatures, yet the genetic position of the causal mutations is not yet known. No temperature sensitive mutants were Cas9 sensitive as well, indicating that the mutation that is triggered by temperature is not relevant to the protein shell that forms around φKZ DNA. 
Escapers of RNA-targeting Cas13a were isolated by plating the mutated and the untreated (wild-type) φKZ phage on bacterial strain SDM078, which was designed with the CRISPR-Cas13a system targeting the head gene gp146 of φKZ. The frequency of escaping the RNA-targeting Cas13a enzyme for wild-type φKZ was determined to be around 0.024 in 106 and around 8 in 106 for the hydroxylamine mutagenized φKZ. The rate of Cas13a escapers was increased to three hundred-fold upon hydroxylamine mutagenesis. 
DNA from seven wild-type φKZ Cas13 escapers and seven mutant φKZ Cas13a escapers from the bacterial strain SDM078 was amplified using PCR. DNA of two wild-type escapers and one mutant escaper was subsequently sequenced. The other 4 wild-type escapers and 6 mutant escapers appeared to have deletions similar to the to-be-sequenced escapers upon PCR amplification. Results of the sequencing indicate that the SDM078 escapers contained various sized deletions that were 399-1026 base pairs long. The deletions always included the protospacer region of the head gene gp146 of the bacteriophage φKZ. 

<!--excerpt-->

| ![](/imgs/systems2.png) | 
|:--:| 
|**Figure 2.** “u” stands for wild-type escapers while “e” stands for the mutagenized escaper phages. In the first group of escapers, the PCR shows the targeted region of the gene (protospacer region), one that no longer exists due to the mutation. The final group of escapers shows a larger section of the DNA. The DNA of the pure φKZ is higher above the rest of the escapers. This shows that there is a larger portion of DNA that remains with the φKZ compared to the escapers which have undergone a deletion (and thus travel further through the PCR gel).

**Conclusions/Discussion**

None of the mutant phages were sensitive to DNA-targeting CRISPR-Cas9 system due to the mutations or deletions likely not occurring in the region in the gene(s) that confer resistance to Cas9 targeting. Since it is known that bacteriophage φKZ occludes the Cas9 enzyme by forming a nucleus-like shell, the hydroxylamine mutagenesis most likely did not result in a weakened shell to allow for DNA cleavage by Cas9. However, in respect to the temperature mutants, inhibition of replication of four mutant phages at a higher temperature suggests that phages were isolated with conditionally lethal mutations in essential genes. 
An increase in frequency of escapers from RNA-targeting enzyme Cas13a upon treatment of bacteriophage φKZ with hydroxylamine shows that hydroxylamine is effective in causing DNA mutations in the gene gp146. However, sequencing results from mutated φKZ escapers of the Cas13a enzyme indicate that the point mutations induced by hydroxylamine are not causing the escaper behavior, but instead, escaper behavior is caused by a deletion of an entire region of the DNA sequence at the gene gp146, possibly created by a genetic mutation. Additionally, the wild-type and the mutagenized escapers both have deletions in the region surrounding the protospacer which indicates that without the protospacer region, the RNA-targeting Cas13a enzyme can no longer target the bacteriophage φKZ. This reveals insight into the specificity of the CRISPR systems.
It is interesting to note the bacteriophage’s ability to escape the Cas13a system, as it targets RNA, an essential part of the phage DNA’s replication system that is not protected by the protein shell. This phenomenon reveals insights into how so much about CRISPR is unknown. This project marks an important discovery in the gene-editing of CRISPR-Cas13a and the first instance of gene-editing with RNA-targeting.
While this project was limited by the manual labor aspect of picking individual plaques, there are methods such as replica plating with a piece of velvet used to transfer phages from one petri dish to the next that could increase the likelihood of finding an escaper with weakened protein shells to allow for the entry of Cas9 enzymes.

**References**

Mendoza, S.D., Nieweglowska, E.S., Govindarajan, S. et al. A bacteriophage nucleus-like compartment shields DNA from CRISPR nucleases. Nature (2019) doi:10.1038/s41586-019-1786-y
R. Villafane (2009); Construction of Phage Mutants; Methods in molecular biology (Clifton, N.J.); 501, 223-37. 
Chaikeeratisak V, Nguyen K, Egan ME, Erb ML, Vavilina A, Pogliano J(2017); “The Phage Nucleus and Tubulin Spindle Are Conserved among Large Pseudomonas Phages”; doi:10.1016/j.celrep.2017.07.064 
Crisprs/Cas9 May Provide New Method for Drug Discovery and Development - Scientific Figure on ResearchGate. Available from: https://www.researchgate.net/figure/Three-different-Types-of-CRISPR-Cas-system-the-CRISPR-Cas-systems-have-been-classified_fig3_303344270 [accessed 3 Mar, 2019]
Chylinski K, Makarova KS, Charpentier E, Koonin EV. Classification and evolution of type II CRISPR-Cas systems. Nucleic Acids Res. 2014;42(10):6091-105.

<!--excerpt-->

